pkg/registry/populator.go

```diff
@@ -63,7 +63,55 @@ func (i *DirectoryPopulator) Populate(mode Mode) error {
 	return nil
 }
 
+func (i *DirectoryPopulator) globalSanityCheck(imagesToAdd []*ImageInput) error {
```

Comment:
Just to clarify - these errors should already be returned by the normal loader, and this PR is specifically to get different return codes for some cases?

This seems fine for now, but IMO we should look at doing something similar to what we plan to do for bundle validation, and output a set of error codes (i.e. in json).

Reply:
+1 on adding that in the future.